r2/r2/tests/unit/lib/utils_test.py

```diff
@@ -95,6 +95,22 @@ def test_coerce_url_to_protocol(self):
             utils.coerce_url_to_protocol('//example.com/foo', 'https'),
             'https://example.com/foo')
 
+    def test_sanitize_url(self):
+        self.assertEquals(
+            utils.sanitize_url('http://dk./'),
+            utils.sanitize_url('http://dk/')
```

Comment:
I think it's probably a better idea to have a static string to compare against - that is, have one of the arguments to `assertEquals` be just `'http://dk/'` (I think? whatever the result is), and have a separate test case covering the more standard inputs.